.circleci/config.yml

```diff
@@ -192,63 +231,50 @@ workflows:
             branches:
               ignore: master
 
-  on_merge_build_test:
+  on_tag_build_release_publish:
```

Comment:
did you change anything in the circle-ci logical flows?
Or just gave each steps better names?